### DISCUSSION TITLE
更新了近期最新版本发生的问题

### DISCUSSION THREAD

**Discussion by Chivent0506** [gkd-kit/gkd#314] [General]
无障碍权限给了，但是就是不跳广告，软件装死了🥹

**Comment by lisonge** [comment]
看起来是这个问题 https://github.com/gkd-kit/gkd/issues/309

**Reply by Chivent0506** [reply]
我也是鸿蒙4.0，那这样我先用旧版本

**Reply by lisonge** [reply]
已更新，可以使用 v1.5.3

**Reply by Chivent0506** [reply]
> 已更新，可以使用 v1.5.3

更啦感谢，已经可以正常使用了
